circuits/main/src/get_account.nr

```diff
@@ -14,22 +14,22 @@ struct Account {
     code_hash: [u8; HASH_LENGTH],
 }
 
-struct AccountWithProof {
-    nonce: u64,
-    balance: u120,
-    state_root: [u8; HASH_LENGTH],
-    code_hash: [u8; HASH_LENGTH],
-    
+struct StateProof {
```

Comment:
This type already exist in `noir-trie-proofs` library - https://github.com/vlayer-xyz/noir-trie-proofs/blob/main/lib/src/lib.nr#L49 we can reuse it

Reply:
Thanks. Fixed

Reply:
Discussed offline that it's better to have custom type here as a protection against lib interface changes